### ISSUE TITLE
Identifiers seems not in the xml file

### ISSUE THREAD

**Issue by gmhhope** [SysBioChalmers/Zebrafish-GEM#5]
### Description of the issue:
Dear Z-GEM developers, 

I am trying to parse the `.xml` files from all your '?-GEM' model. I found that in Zebrafish-GEM, the identifiers showed in the `metabolites.tsv` may not be in the `.xml` metabolite entries. 

I am using `cobra` to parse the model `model = cobra.io.read_sbml_model(xmlFile)`

This results in:
```
'' is not a valid SBML 'SId'.
https://identifiers.org/taxonomy/ does not conform to 'http(s)://identifiers.org/collection/id' or'http(s)://identifiers.org/COLLECTION:id
```
When I look at the annotation field, there seems to only have the model ID remained the same in all metabolite entries:
`model.metabolites[5].__dict__`
```
{'_id': 'MAM00003l',
 'name': '(10Z)-heptadecenoic acid',
 'notes': {},
 '_annotation': {'sbo': 'SBO:0000247'},
 '_model': <Model  at 0x7f8f52e934f0>,
 '_reaction': {<Reaction MAR03549 at 0x7f8f7055cc70>,
  <Reaction MAR09738 at 0x7f8f72624fa0>},
 'formula': 'C17H31O2',
 'compartment': 'l',
 'charge': -1,
 '_bound': 0.0}
```

#### Expected feature/value/output:
I would expect to see the identifiers in `metabolite.tsv` populated or synchronized with the `.xml` files. 

And can I expect the `MAM` IDs is linkable between different GEM models?

#### Reproducing these results:

```python
# https://cobrapy.readthedocs.io/en/latest/io.html#SBML
import cobra
import os
import requests

# Human-GEM e.g.,
model_name = 'Zebrafish-GEM'

outputfdr = f"../../output/ATLAS_collection/{model_name}"
try:
    os.makedirs(outputfdr)
except:
        None

BRANCH = 'main'
XML = os.path.join(outputfdr,f'{model_name}.xml')
with open(XML, 'w') as f:
    r = requests.get(f'https://github.com/SysBioChalmers/{model_name}/blob/' + BRANCH + f'/model/{model_name}.xml?raw=true')
    f.write(r.text)

with open(XML, 'r') as f:
    xmlFile = f.read()

model = cobra.io.read_sbml_model(xmlFile)
model.metabolites[5].__dict__

```

Hope you can help! Thanks very much!

Thanks,
Minghao Gong
The Jackson Laboratory for Genomic Medicine

**Comment by gmhhope** [comment]
Also attached is my pased table, which shows probably only `inchikey` is present in the annotation field
[Zebrafish-GEM_listOfMet.csv](https://github.com/SysBioChalmers/Zebrafish-GEM/files/8246298/Zebrafish-GEM_listOfMet.csv)

Thanks,
MG

**Comment by haowang-bioinfo** [comment]
this has been fixed in v1.2.0, feel free to reopen if problem stays
